src/app/rosetta/lib/block.ml

```diff
@@ -479,8 +448,8 @@ WITH RECURSIVE chain AS (
 
     let query =
       Caqti_request.collect Caqti_type.int typ
-        {| SELECT DISTINCT ON (i.hash,i.type,bic.sequence_no,bic.secondary_sequence_no) i.id, i.type, i.receiver_id, i.fee, i.token, i.hash,
-            bic.receiver_account_creation_fee_paid, pk.value as receiver,
+        {| SELECT DISTINCT ON (i.hash,i.typ,bic.sequence_no,bic.secondary_sequence_no) i.id, i.typ, i.receiver_id, i.fee, i.hash,
```

Comment:
`command_type` ?

Reply:
Updated in [cdd78a3](https://github.com/MinaProtocol/mina/pull/11960/commits/cdd78a3f2f00841fa23a2f32d563b618b4db5103).